src/YBarGraph.h

```diff
@@ -32,7 +32,11 @@
 class YBarGraphPrivate;
 class YBarGraphSegment;
 
-
+/**
+ * A graph showing partitioning of a whole.
+ * The whole is divided into YBarGraph segments,
+ * each of which has a relative size, a color, and a label.
```

Comment:
I think link to class below would be nice and there is also inconsistency as below you mention two colors ( label and background) and here only generic color.